### PR TITLE
S85629 increased stack trace limit to 5000 to capture more stack frames

### PR DESCRIPTION
**Pull request by sboles**

### CODE REVIEW

**Comment by sboles** [comment]
@pbomb We'd like to increase the size of the stack data captured. Also, does this require a version bump?

**Comment by pbomb** [comment]
If we make a change in client metrics, we do need a version bump and need AppSDK to specify this new version. Can we make this change use a property that gets passed in instead of hard-coding the limit in this library? That might give us more flexibility to make changes without as much effort - even toggling on different values if we need to experiment.

**Comment by sboles** [comment]
@pbomb Any issues w/ the updates to take a config?

**Comment by pbomb** [comment]
Oh, I didn't see this update. It looks good now. :ship it:

File: src/windowErrorListener.js [hunk 1]
```diff
@@ -14,10 +14,16 @@
...
      * @param {RallyMetrics.ClientMetricsAggregator} aggregator
      * @param {Boolean} [supportsOnError=true] Does the browser support window.onerror?
+     * @param {Object} config Configuration object
+     * @param {Number} [config.stackLimit] If defined, the stack trace for the error will be truncated to this limit
      */
-    var ErrorListener = function(aggregator, supportsOnError) {
+    var ErrorListener = function(aggregator, supportsOnError, config) {
         var useOnError = _.isBoolean(supportsOnError) ? supportsOnError : browserSupportsOnError;
         this.aggregator = aggregator;
+        this._stackLimit = null;
```
Comment (robertfwest) [inline src/windowErrorListener.js]: If the caller did not provide a stackLimit config value, we will not get _any_ stack info. Seems to me that we should default to Error.stackTraceLimit if defined and 10 if not.

Comment (sboles) [inline src/windowErrorListener.js]: won't you get the full stack vs a truncated one?

Comment (robertfwest) [inline src/windowErrorListener.js]: Shoot. I totally ignored line 55. Mea culpa.

Comment (sboles) [inline src/windowErrorListener.js]: sweet, shippin

**Comment by pbomb** [comment]
:shipit:

**Pull request merged**